critiquebrainz/db/review.py

```diff
@@ -306,12 +272,8 @@ def create(*, release_group=None, entity_id=None,
             "license": dict,
         }
     """
-    if release_group:
-       entity_id = release_group
-       entity_type = "release_group"
-    else:
-        if not entity_type or not entity_id:
-          raise TypeError("Entity Type or ID not defined")
+    if language not in supported_languages:
+        raise TypeError("Language: {} is not supported".format(language))
```

Comment:
This doesn't look like a `TypeError`. Take a look its description: https://docs.python.org/3.6/library/exceptions.html#TypeError. Someone might pass an invalid language as a string.

[`ValueError`](https://docs.python.org/3.6/library/exceptions.html#ValueError) would be the right one to use here.